Views/JSuPCB/Edit.cshtml

```diff
@@ -0,0 +1,67 @@
+@model SUMBER.Models.Sumber.JSuPCB
+
+@{
+    ViewData["Title"] = "PCB - Ubah";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<partial name="_JSEditradio" />
```

Comment:
Change name '_JSEditradio' to '_JSEdit'.
Reason: All javascript will be coded here. Not only for the radio button. 'Edit' is used as reference to the Edit.cshtml file in the corresponding Model of the View folder.